src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
```

Comment:
Avoid using the literal '3' and '20' directly in the expression. Define constants that represent the minimum days for a short-term rental and the corresponding discount.